src/pretalx/orga/templates/orga/cfp/question_form.html

```diff
@@ -64,6 +64,11 @@ <h2>
             {% bootstrap_field form.tracks layout='event' %}
         </span>
         {% endif %}
+        {% if form.submission_types %}
+        <span id="limit-submission">
```

Comment:
Let's change this to a `class` here and five lines above, because `id`s are technically meant to be unique.